script/server/cli.ts

```diff
@@ -2,8 +2,10 @@
 import { argv } from 'yargs';
 import server from '.';
 
-server({port: parseInt(process.env.PORT || '', 10), ...argv})
-  .then(({port}) => console.log(`running on port: ${port}`))
+const port = parseInt(process.env.PORT || '0', 10);
```

Comment:
it seems like maybe we don't even need this here? because https://github.com/openstax/rex-web/pull/39/files#diff-3f7ce2d17b676da9527903e740774c69R16 is reading out of the env for the defaultOption